src/views/EdgeNode/ListView.vue

```diff
@@ -24,7 +24,7 @@
     }
   })
 
-  let hasContentToList = true
+  let hasContentToList = false
```

Comment:
Me faz mais sentido ser uma ref devido ao v-if no template, pois assim o vue possui melhor controle.
```suggestion
  const hasContentToList = ref(false)
```

Reply:
Não consigo apontar outras sugestões aqui referentes à essa, mas se caso adotar o uso do ref, alterar tbm a atribuição na linha 74.